pkgs/development/haskell-modules/configuration-common.nix

```
@@ -549,24 +549,24 @@ self: super:
 
   # Manually maintained
   cachix-api = overrideCabal (drv: {
-    version = "1.7.7";
+    version = "1.7.8";
     src = pkgs.fetchFromGitHub {
       owner = "cachix";
       repo = "cachix";
-      rev = "v1.7.7";
-      hash = "sha256-D0gP8srrX0qj+wNYNPdtVJsQuFzIng3q43thnHXQ/es=";
+      rev = "v1.7.8";
```

```suggestion
      tag = "v1.7.8";
```

https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub